### ISSUE TITLE
link to "What's new" in the about section.

### ISSUE THREAD

**Issue by billytwotone** [ChrisTitusTech/winutil#2325]
Now that Winutil has a lot of users, a link to "What's new" would be useful. This link would provide details of improvements and changes for each version. Obviously users will be interested to test out what is new, instead of not knowing about the new features until they accidentally stumble upon something new.

The link to "What's new" would conveniently go in the "About section". Click on the cog in the right-hand upper corner, and find the "What's new" link with the other "About" information.

**Comment by og-mrk** [comment]
@MyDrift-user I think what the author of this issue meant is, have something like a Change Log File, which can be accessed by simply clicking on a `What's New` button in a Drop-Down Menu Item, which'll be helpful to users who use WinUtil often, or are interested in its development progress.

Should be handled in the next patch series in #2226

**Comment by MyDrift-user** [comment]
@og-mrk Doesn't the release have the changelog as description? I thaught that that is named changelog

**Comment by og-mrk** [comment]
> @og-mrk Doesn't the release have the changelog as description? I thaught that that is named changelog

... I guess a changelog can contain both the detailed changes in a release, plus a short/nice summery of things that might interests users, like New Tweaks to see, Fixing Known bugs that effect a lot of users, and UI changes.

**Comment by billytwotone** [comment]
Thank you for your replies. I have clicked on the date, and I have found the summery of the changes. This sort of summery is fine for me, but it is not intuitively obvious that the date was a link. Maybe, one of those question marks placed adjacent to the date that reveals explanatory pop-up text would suffice to make it obvious. I am sure that a lot of users would have missed that the date was a link to "What's new". 

I would think that a link to the detailed code changes would not really be of interest to most users, and would not be needed on the user interface. This sort of detail would be available on the Github page, for those who are interested in seeing the code changes, is it not?
